test/integration/partials-test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
+const path = require('path');
+const dist = path.resolve(__dirname, '..', '..', 'dist');
+
+describe('Partials test', function() {
```

Comment:
```suggestion
describe('Service worker partials', function() {
```

Reply:
I realize I used "Build test" in the other file but was thinking we probably want to rename that to just "Build".